docs/guides/payments.rst

```diff
@@ -178,52 +164,74 @@ Example
+| name                    | type     | description                                                                        |
++-------------------------+----------+------------------------------------------------------------------------------------+
+| ``payment_information`` | ``dict`` | Payment information, containing the tokens, amount, currency and billing.          |
++-------------------------+----------+------------------------------------------------------------------------------------+
+| ``connection_params``   | ``dict`` | List of parameters used for connecting to the payment's gateway.                   |
++-------------------------+----------+------------------------------------------------------------------------------------+
 
 Returns
 ^^^^^^^
 
-+------------------+-----------------+-----------------------------------------------------------------------------------------------------------+
-| name             | type            | description                                                                                               |
-+------------------+-----------------+-----------------------------------------------------------------------------------------------------------+
-| ``txn``          | ``Transaction`` | Transaction created during the payment process, with ``is_success`` set to ``True`` if no error occurred. |
-+------------------+-----------------+-----------------------------------------------------------------------------------------------------------+
-| ``error``        | ``str``         | Error message to be displayed in the UI, empty if no error occurred.                                      |
-+------------------+-----------------+-----------------------------------------------------------------------------------------------------------+
-| ``client_token`` | ``str``         | Unique client's token that will be used as his indentifier in the payment process.                        |
-+------------------+-----------------+-----------------------------------------------------------------------------------------------------------+
++----------------------+----------------------------+------------------------------------------------------------------------------------------------------------------------------------------+
+| name                 | type                       | description                                                                                                                              |
++----------------------+----------------------------+------------------------------------------------------------------------------------------------------------------------------------------+
+| ``gateway_response`` | ``dict`` or ``list[dict]`` | Dictionary or list of dictionaries containing details about every transaction, with ``is_success`` set to ``True`` if no error occurred. |
```

Comment:
What is the required fields of ```gateway_response``` which is going to be validated in ```validate_gateway_response```? It worths to mention it in the docs.